files/besu/smart_contracts/package.json

```diff
@@ -8,10 +8,11 @@
   },
   "license": "Apache-2.0",
   "dependencies": {
+    "async-promise-pool": "^1.0.3",
+    "ethereumjs-tx": "1.3.7",
+    "fs-extra": "^10.0.0",
     "solc": "^0.7.0",
     "web3": "^1.2.11",
-    "web3-eea": "^0.9.0",
-    "ethereumjs-tx": "1.3.7",
-    "fs-extra": "^10.0.0"
+    "web3-eea": "^0.9.0"
```

Comment:
should we use the new web3js-quorum here?

Reply:
Is it cross compatible with besu or just quorum specific?